src/api/chrome.js

```diff
@@ -215,3 +218,32 @@ const getBadgeCounter = async (initialValue = 0) => {
   }
   return badgeCounter
 }
+
+/**
+ * Retrieves current tab url and validates it. Returns valid tab url otherwise undefined.
+ * @returns the tab url or undefined
+ */
+
+export const getCurrentTabUrlBackground = async () => {
+  let queryOptions = { active: true }
```

Comment:
never use let unless you intend to mutate the variable (which is almost never the case)

Reply:
Yeah copy & paste ...  good catch 👍